test/ng/directive/booleanAttrsSpec.js

```diff
@@ -118,221 +118,3 @@ describe('boolean attr directives', function() {
     }));
   });
 });
-
-
-describe('ngSrc', function() {
```

Comment:
All these tests here and below were in the wrong file. I have moved them to their appropriate places.